azure-mgmt-graph-rbac/src/main/java/com/microsoft/azure/management/graphrbac/implementation/GraphRbacManager.java

```diff
@@ -8,31 +8,41 @@
 
 import com.microsoft.azure.AzureEnvironment;
 import com.microsoft.azure.AzureResponseBuilder;
-import com.microsoft.azure.management.resources.fluentcore.utils.ProviderRegistrationInterceptor;
-import com.microsoft.azure.serializer.AzureJacksonAdapter;
-import com.microsoft.rest.interceptors.RequestIdHeaderInterceptor;
-import com.microsoft.rest.RestClient;
 import com.microsoft.azure.credentials.AzureTokenCredentials;
 import com.microsoft.azure.management.apigeneration.Beta;
+import com.microsoft.azure.management.graphrbac.Applications;
+import com.microsoft.azure.management.graphrbac.Groups;
 import com.microsoft.azure.management.graphrbac.ServicePrincipals;
 import com.microsoft.azure.management.graphrbac.Users;
 import com.microsoft.azure.management.resources.fluentcore.arm.AzureConfigurable;
 import com.microsoft.azure.management.resources.fluentcore.arm.implementation.AzureConfigurableImpl;
+import com.microsoft.azure.management.resources.fluentcore.model.HasInner;
+import com.microsoft.azure.management.resources.fluentcore.utils.ProviderRegistrationInterceptor;
+import com.microsoft.azure.serializer.AzureJacksonAdapter;
+import com.microsoft.rest.RestClient;
+import com.microsoft.rest.interceptors.RequestIdHeaderInterceptor;
 
 /**
  * Entry point to Azure Graph RBAC management.
  */
 @Beta
-public final class GraphRbacManager {
+public final class GraphRbacManager implements HasInner<GraphRbacManagementClientImpl> {
     private String tenantId;
     // The sdk clients
     private final GraphRbacManagementClientImpl graphRbacManagementClient;
     // The collections
     private Users users;
+    private Groups groups;
     private ServicePrincipals servicePrincipals;
+    private Applications applications;
+
+    @Override
+    public GraphRbacManagementClientImpl inner() {
+        return graphRbacManagementClient;
+    }
 
     /**
-     * Creates an instance of GraphRbacManager that exposes resource management API entry points.
+     * Creates an instance of GraphRbacManager that exposes graph rbac management API entry points.
```

Comment:
capitalization "Graph" is a proper name; RBAC is an acronym so "Graph RBAC" in javadocs